README.md

```diff
@@ -159,6 +159,10 @@ RMT is replacing some functionality of [SMT](https://github.com/SUSE/smt). Follo
 <a name="staging">1</a>: Functionality is offered by [SUSE Manager](https://www.suse.com/documentation/suse-best-practices/susemanager/data/susemanager.html)  
 <a name="res">2</a>: RES support is planned for SLES15 SP1
 
+## API documentation
+
+The documentation about the implemented SCC API can be found [here](https://github.com/SUSE/connect/blob/master/doc/SCC-API-(Implemented).md).
```

Comment:
Could it say something like `RMT partially implements [SUSE Customer Center API](link)`?

Reply:
sure